eogrow/utils/testing.py

```diff
@@ -183,9 +187,10 @@ def _rounder(x: float, y: float) -> tuple[float, float]:
 
     if len(gdf):
         subsample: gpd.GeoDataFrame = gdf.sample(min(len(gdf), config.num_random_values), random_state=42)
-        subsample["centroid"] = subsample.centroid.apply(lambda point: _rounder(*point.coords[0]))
+        subsample["centroid"] = subsample.centroid.apply(lambda point: _rounder(point.coords[0]))
```

Comment:
With the change to `_rounder` proposed above, this could then be 

```suggestion
        subsample["centroid"] = subsample.centroid.apply(_rounder)
```